Content.Shared/Preferences/Appearance/HumanoidCharacterAppearance.cs

```diff
@@ -66,5 +67,54 @@ public static class HumanoidVisualLayersExtension
                 _ => throw new ArgumentOutOfRangeException()
             };
         }
+
+
+        /// <summary>
+        /// Returns the corresponding Layer including sublayers
+        /// </summary>
+        /// <param name="part"></param>
+        /// <returns></returns>
+        public static IReadOnlyCollection<HumanoidVisualLayers> ToHumanoidLayers(this IBodyPart part)
```

Comment:
Since you are only iterating over this you can make it return an `IEnumerable<HumanoidVisualLayers>`, making each case `yield return` each layer and using `yield break` in empty ones.